src/database/models/userDetails.model.js

```diff
@@ -0,0 +1,51 @@
+import { Sequelize, DataTypes } from 'sequelize';
+import sequelize from '../config/db';
+
+const UserDetailsModel = sequelize.define('user_details', {
+  id: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+    unique: true,
+    autoIncrement: true,
+  },
+  userId: {
+    type: DataTypes.UUID,
+    allowNull: false,
+    unique: true,
+    primaryKey: true,
```

Comment:
`id `is the one to be primary Key not `userId`

Reply:
no, here i have to, and nothing it changes there